package.json

```diff
@@ -38,14 +36,8 @@
     "typescript": "^4.4.2"
   },
   "dependencies": {
-    "@types/nodemailer": "^6.4.4",
-    "@types/nodemailer-direct-transport": "^1.0.32",
-    "@types/nodemailer-smtp-transport": "^2.7.5",
     "aws-sdk": "^2.999.0",
-    "bcrypt": "^5.0.1",
-    "bcryptjs": "^2.4.3",
```

Comment:
why is this removed?

Reply:
They were imports that weren't being used

Reply:
Oh they are used.....

Reply:
server/graphql/schema/mutations/createUser.ts?

Reply:
added them back